osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -34,19 +36,24 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
             var osuLastLastObj = (OsuDifficultyHitObject)current.Previous(1);
 
             // Calculate the velocity to the current hitobject, which starts with a base distance / time assuming the last object is a hitcircle.
-            double currVelocity = osuCurrObj.LazyJumpDistance / osuCurrObj.StrainTime;
+            double currVelocity = osuCurrObj.JumpDistance / osuCurrObj.StrainTime;
+
+            // Bandaid to prevent Lost Umbrella from gaining
+            double sliderJumpsAdjustingMultiplier = 1.0;
 
             // But if the last object is a slider, then we extend the travel velocity through the slider into the current object.
             if (osuLastObj.BaseObject is Slider && withSliderTravelDistance)
             {
                 double travelVelocity = osuLastObj.TravelDistance / osuLastObj.TravelTime; // calculate the slider velocity from slider head to slider end.
                 double movementVelocity = osuCurrObj.MinimumJumpDistance / osuCurrObj.MinimumJumpTime; // calculate the movement velocity from slider end to current object
 
-                currVelocity = Math.Max(currVelocity, movementVelocity + travelVelocity); // take the larger total combined velocity.
+                double newVelocity = Math.Max(currVelocity, movementVelocity + travelVelocity); // take the larger total combined velocity.
+                if (currVelocity > 0) sliderJumpsAdjustingMultiplier = currVelocity / newVelocity;
+                currVelocity = newVelocity;
```

Comment:
```suggestion
                double newVelocity = Math.Max(currVelocity, movementVelocity + travelVelocity); // take the larger total combined velocity.

                if (currVelocity > 0)
                    sliderJumpsAdjustingMultiplier = currVelocity / newVelocity;

                currVelocity = newVelocity;
```

This really confused me at first read, please make it a little clearer like so..